File: Easy/Palindrome-Number/Palindrome Number.js
/*
9. Palindrome Number
https://leetcode.com/problems/palindrome-number/
Difficulty: Easy

Determine whether an integer is a palindrome. Do this without extra space.

@param {number} x
@return {boolean}
*/
var isPalindrome = function(x) {
    var P=true;
    var n=x.toString().length;
    for (var i=0; i<n; i++){
        if (x.toString().charAt(i)!=x.toString().charAt(n-(i+1))){
            P=false;
        }
    }
    return P;
};

File: Easy/Palindrome-Number/Palindrome Number.java
/*
9. Palindrome Number
https://leetcode.com/problems/palindrome-number/
Difficulty: Easy

Determine whether an integer is a palindrome. Do this without extra space.
*/

public class Solution {
    public boolean isPalindrome(int x) {
        boolean palindrome=true;
        char [] numbers = String.valueOf(x).toCharArray();
        for (int i=0; i<numbers.length-1; i++){
            if (numbers[i]!=numbers[numbers.length-(i+1)]){
                palindrome=false;
            }
        }
    return palindrome;
    }
}
